### PR TITLE
Paraunit as standalone PHAR archive

### PR DESCRIPTION
**Pull request by taueres**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ._*
 *.tgz
 *.phar
+.idea
```
Comment (Jean85) [inline .gitignore]: Please add this to your global .gitIgnore and remove it from the project, it's user specific.

File: phar/stub.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+if (isset($argv[1]) && $argv[1] === 'phpunit') {
+    // Do not run Paraunit itself, actually run the embedded PHPUnit
+    // This is required for running PHPUnit parallel subprocesses
+    array_shift($argv);
+    define('__PHPUNIT_PHAR__', str_replace(DIRECTORY_SEPARATOR, '/', __FILE__));
+    define('__PHPUNIT_PHAR_ROOT__', 'phar://paraunit.phar');
+    PHPUnit_TextUI_Command::main();
+}
+require 'phar://paraunit.phar/src/Paraunit/Bin/run-paraunit.inc.php';
+__HALT_COMPILER();
```
Comment (Jean85) [inline phar/stub.php]: New line missing at the end of file

**Comment by Jean85** [comment]
Apart for the minor comments, LGTM! :+1: 
Good work!

**Comment by jwage** [comment]
:+1: I found [box](https://github.com/box-project/box2) to  be really nice for generating phar files. See https://github.com/jwage/phpchunkit/blob/master/box.json for an example box.json for jwage/phpchunkit.

**Comment by Jean85** [comment]
Thanks for the suggestion @jwage ! I'll take a look at it ASAP

**Comment by taueres** [comment]
Hopefully, I'll have a look to this PR during this weekend. If I recall correctly, only minor changes are needed. With the Box package we could remove those ugly shell scripts. I'll keep you updated!

**Comment by Jean85** [comment]
Thanks @taueres !

**Comment by Jean85** [comment]
Closing this in favor of #62

**Pull request closed without merging**